gemfiles/Gemfile.rails-7.0

```diff
@@ -2,6 +2,6 @@
 
 source 'https://rubygems.org'
 
-gem 'rails', '~> 7.0.2.2'
+gem 'rails', '~> 7.0.8'
```

Comment:
The two changes to the existing gemfiles look good. However, they are not related to adding official support for Rails 7.1, and I'd prefer to have them in a separat PR.